echo-pipelinetriggers/src/main/java/com/netflix/spinnaker/echo/pipelinetriggers/monitor/WebhookEventMonitor.java

```diff
@@ -0,0 +1,140 @@
+/*
+ * Copyright 2016 Google, Inc.
```

Comment:
I read that as "but [I] am just checking what we need",
That sounds like a task you're undertaking. I'll await your response on that.